cmd/cli/main.go

```diff
@@ -47,7 +51,30 @@ func checkHubAddressIsSet(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+// commandError allow to present any internal error as JSON
+type commandError struct {
+	Err     error  `json:"error"`
+	Message string `json:"message"`
+}
+
+func (ce *commandError) ToJSONString() string {
+	j, _ := json.Marshal(ce)
```

Comment:
I haven't ever meet that behaviour.
I can use little hack with unmarshallable error field and marshallable string field, like this:
https://play.golang.org/p/FtYxrFBiJj